File: 프로그래머스/고득점Kit-스택큐/주식가격.py
def solution(prices):
    answer = []
    for i in range(len(prices)):
        cur=prices[i]
        time=0
        for j in range(i+1,len(prices)):
            if cur<=prices[j]:
                time+=1
            else:
                time+=1
                break
        answer.append(time)
        
    return answer

File: milla/주식가격.cpp
#include <string>
#include <vector>
#include <stack>

using namespace std;

vector<int> solution(vector<int> prices) {
    vector<int> answer;
    
    for(int i=0;i<prices.size();i++){
        int target = prices[i];
        //stack<int> st;
        int tmp=0;
        for (int j=i+1;j<prices.size();j++){
            tmp++;
            if(prices[j]<target){
                break;
            }
        }
        answer.push_back(tmp);
    }
    
    return answer;
}